R/pkg/R/SQLContext.R

```diff
@@ -75,8 +75,8 @@ infer_type <- function(x) {
 #'\dontrun{
 #' sc <- sparkR.init()
 #' sqlContext <- sparkRSQL.init(sc)
-#' rdd <- lapply(parallelize(sc, 1:10), function(x) list(a=x, b=as.character(x)))
-#' df <- createDataFrame(sqlContext, rdd)
+#' df1 <- as.DataFrame(sqlContext, iris)
```

Comment:
`iris` is just a dataset present by default on all R installations. I think its fine to use that.

Also I think its better to change the old examples as they use the RDD API which is not public.